.travis/publish_docs.sh

```diff
@@ -88,13 +102,21 @@ cp -r target/staging/* "$release_tag"/
 
 printf "${CYAN}Creating redirecting index.html.$RESET \n"
 mv target/build_index.sh "."
-sh ./build_index.sh "$release_tag"
+bash ./build_index.sh "$release_tag"
```

Comment:
This explanation doesn't make sense to me. `sh` should be even more primitive than `bash`.